private/bufpkg/bufconnect/interceptors_test.go

```diff
@@ -104,24 +104,42 @@ func TestNewAuthorizationInterceptorProvider(t *testing.T) {
 
 func TestNewTokenSetFromEnv(t *testing.T) {
 	tokenSet, err := NewTokenProviderFromContainer(app.NewEnvContainer(map[string]string{
-		tokenEnvKey: "default,user:token@remote",
+		tokenEnvKey: "default",
 	}))
 	assert.NoError(t, err)
 	token := tokenSet.RemoteToken("fake")
 	assert.True(t, tokenSet.IsFromEnvVar())
 	assert.Equal(t, "default", token)
-	token = tokenSet.RemoteToken("remote")
-	assert.Equal(t, "token", token)
 }
 
 func TestNewTokenSetFromString(t *testing.T) {
 	_, err := NewTokenProviderFromString("default")
 	assert.NoError(t, err)
-	_, err = NewTokenProviderFromString("user1:token1@remote,user2:token2@remote")
+	_, err = NewTokenProviderFromString("token1@host1")
```

Comment:
tests should include verifying the values from the returned token provider, ie `assert.Equal(t, "token1", tokenProvider.RemoteToken("host1")`